src/styles/Book.module.css

```
@@ -64,4 +64,39 @@
 
 .iframeSlideOutRight {
   animation: slideOutToRight 0.5s forwards;
-}
\ No newline at end of file
+}
+
+.notebook {
```

Add a border radius:
```suggestion
.notebook {
   border-radius:15px;
```